src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " started her work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getName() + " finished my work.");
```

Comment:
The message should be more informative and indicate the type of machine that has stopped working. For example, 'Excavator finished its work.' Also, ensure consistency in the use of pronouns; 'her' is used in the start message and 'my' in the stop message.